setup.py

```diff
@@ -104,6 +104,7 @@ def get_mpi_flags():
     "testing": ["testflo>=1.4.7"],
     "docs": ["sphinx", "breathe", "sphinxcontrib-programoutput"],
     "mphys": ["mphys>=1.1.0", "openmdao>=3.25.0"],
+    "caps2tacs": ["funtofem>=0.3.2", "imageio>=2.16.1"],
```

Comment:
Not sure if funtofem should really be included as a depency when using caps2tacs. It can run without it, but it does have some additional features if it is available (uses importlib to run without it).

Reply:
@sean-engelstad Let's leave it off the list. As long as there is a guard to prevent an `ImportError`, I'm alright with it

Reply:
@timryanb I removed the funtofem dependency here on the TACS side. Funtofem is modular in that it uses TACS but only through importlib guards. So I think we're good. The funtofem setup.py does not mention tacs.